src/main/resources/i18n/messages.properties

```diff
@@ -286,4 +286,20 @@ UNION_SQL_INJECTION_URL_PARAM_REMOVES_SINGLE_QUOTE_WITH_SINGLE_QUOTE_APPENDED_TO
  hence \"Union\" keyword can be used to club the results and extract the application data.
  
 BLIND_SQL_INJECTION_URL_PARAM_APPENDED_DIRECTLY_TO_QUERY=Query param is directly appended to the Query hence Query can be manipulated.
-BLIND_SQL_INJECTION_URL_PARAM_WRAPPED_WITH_SINGLE_QUOTE_APPENDED_TO_QUERY=Query param is wrapped around "'" and then appended to the SQL Query hence query can be manipulated.
\ No newline at end of file
+BLIND_SQL_INJECTION_URL_PARAM_WRAPPED_WITH_SINGLE_QUOTE_APPENDED_TO_QUERY=Query param is wrapped around "'" and then appended to the SQL Query hence query can be manipulated.
+
+#### SSRF Vulnerability
+
+SSRF_VULNERABILITY=In a Server-Side Request Forgery (SSRF) attack, the attacker can abuse functionality on the server to \
+read or update internal resources.  The attacker can supply or modify a <strong>URL</strong> which the code running on the server will read \
+or submit data to, and by carefully selecting the URLs, <strong>the attacker may be able to read server </strong>  \
+configuration such as AWS metadata, connect to internal services like http enabled databases or perform post requests \
+towards internal services which are not intended to be exposed. \
+Important Links on SSRF : \
+<ol> \
+ <li> <a href="https://en.wikipedia.org/wiki/Server-side_request_forgery" target="_blank">Wiki Link</a>  \
+ <li> <a href="https://owasp.org/www-community/attacks/Server_Side_Request_Forgery" target="_blank">Owasp SSRF</a> \
+ <li> <a href="https://www.youtube.com/watch?v=AsPpxqIlTDU" target="_blank">SSRF (Server Side Request Forgery) by Musab Khan</a>\
+ </ol>
+IMAGE_URL_PASSED_TO_REQUEST=The URL that the user enters is directly passed to a request.
```

Comment:
can you please append the vulnerability name before the Message Key such that we can avoid the conflicts.